<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>881</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>431</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="app_res.qrc">:/resources/resources/phone_status_bar.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="app_res.qrc">:/resources/resources/arrow.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>70</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="app_res.qrc">:/resources/resources/home.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="app_res.qrc">:/resources/resources/arrow.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#d3311b;&quot;&gt;Post Car Listing&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>20</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(235, 235, 235);</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>15</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#d3311b;&quot;&gt;Post Car Listing&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>15</y>
       <width>67</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="app_res.qrc">:/resources/resources/user_circular.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="back_button">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>11</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>235</red>
           <green>235</green>
           <blue>235</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>211</red>
           <green>49</green>
           <blue>27</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>235</red>
           <green>235</green>
           <blue>235</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>235</red>
           <green>235</green>
           <blue>235</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>235</red>
           <green>235</green>
           <blue>235</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>211</red>
           <green>49</green>
           <blue>27</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>235</red>
           <green>235</green>
           <blue>235</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>235</red>
           <green>235</green>
           <blue>235</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>235</red>
           <green>235</green>
           <blue>235</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>168</red>
           <green>169</green>
           <blue>170</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>235</red>
           <green>235</green>
           <blue>235</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>235</red>
           <green>235</green>
           <blue>235</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(235, 235, 235);</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>17</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="app_res.qrc">:/resources/resources/leftarrow.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#d3311b;&quot;&gt;Listing Preview&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>178</x>
      <y>75</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#d3311b;&quot;&gt;. . .&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="app_res.qrc">:/resources/resources/arrow.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="post_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>800</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>92</red>
          <green>184</green>
          <blue>92</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>92</red>
          <green>184</green>
          <blue>92</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>92</red>
          <green>184</green>
          <blue>92</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>92</red>
          <green>184</green>
          <blue>92</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>92</red>
          <green>184</green>
          <blue>92</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>92</red>
          <green>184</green>
          <blue>92</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>160</red>
          <green>161</green>
          <blue>163</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>92</red>
          <green>184</green>
          <blue>92</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>170</red>
          <green>171</green>
          <blue>172</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>127</red>
          <green>127</green>
          <blue>127</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>92</red>
          <green>184</green>
          <blue>92</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>92</red>
          <green>184</green>
          <blue>92</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>35</red>
          <green>38</green>
          <blue>41</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(92, 184, 92);</string>
    </property>
    <property name="text">
     <string>Accept Terms and Post Listing</string>
    </property>
   </widget>
   <widget class="QPushButton" name="next_image_button">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>180</y>
      <width>41</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prev_image_button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>41</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="view_3d_button">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>257</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>3D View</string>
    </property>
   </widget>
   <widget class="QLabel" name="listing_title_bar_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>250</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#d3311b;&quot;&gt;Car Details&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="listing_title_bar_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>470</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#d3311b;&quot;&gt;Car Description&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="listing_title_bar_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>570</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#d3311b;&quot;&gt;Car Location&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QQuickWidget" name="quickWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>600</y>
      <width>411</width>
      <height>201</height>
     </rect>
    </property>
    <property name="resizeMode">
     <enum>QQuickWidget::SizeRootObjectToView</enum>
    </property>
    <property name="source">
     <url>
      <string>qrc:/resources/qml/map.qml</string>
     </url>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>800</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="app_res.qrc">:/resources/resources/white_checkmark.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="car_descr_box">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>490</y>
      <width>401</width>
      <height>81</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="car_info_table">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>401</width>
      <height>192</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>Category</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Company</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Model</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Year</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Mileage (Km)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Condition</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Engine (cc)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Power (bhp)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Transmission</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Fuel Type</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>City Fuel Consumption</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Motorway Fuel Consumption</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Color</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Interior Color</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string># Doors</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Registration Plate</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Price</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="listing_title_bar">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#d3311b;&quot;&gt;Listing Title&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="app_res.qrc"/>
 </resources>
 <connections/>
</ui>
